lib/src/conflicts.rs

```diff
@@ -58,6 +58,15 @@ pub const MIN_CONFLICT_MARKER_LEN: usize = 7;
 /// existing markers.
 const CONFLICT_MARKER_LEN_INCREMENT: usize = 4;
 
+/// Marker for missing terminating newline in a term of a conflict.
+const NO_EOL_MARKER: &str = " [noeol]";
+
+/// Marker for missing terminating newline in the "add" side of a diff.
+const ADD_NO_EOL_MARKER: &str = " [+noeol]";
+
+/// Marker for missing terminating newline in the "remove" side of a diff.
+const REMOVE_NO_EOL_MARKER: &str = " [-noeol]";
```

Comment:
nit: since these strings are not meant to be parsed, I wouldn't call them as `MARKER`. Maybe `COMMENT` or `LABEL` is better?